ui/page/root/wallet_list.go

```diff
@@ -244,12 +244,12 @@ func (pg *WalletSelectorPage) badWalletsWrapper(gtx C, badWalletsList []*badWall
 
 func (pg *WalletSelectorPage) walletWrapper(gtx C, item *load.WalletItem) D {
 	return cryptomaterial.LinearLayout{
-		Width:      cryptomaterial.WrapContent,
-		Height:     cryptomaterial.WrapContent,
-		Padding:    layout.UniformInset(values.MarginPadding16),
-		Background: pg.Theme.Color.Surface,
-		Alignment:  layout.Middle,
-		Shadow:     pg.shadowBox,
+		Width:       cryptomaterial.WrapContent,
+		Height:      cryptomaterial.WrapContent,
+		Padding:     layout.UniformInset(values.MarginPadding16),
+		Background:  pg.Theme.Color.Surface,
+		Shadow:      pg.shadowBox,
+		Orientation: layout.Vertical,
 		Margin: layout.Inset{
```

Comment:
These padding and margin values are not consistent with what I see on figma.
*This can probably be made into a separate issue since these values weren't added or modified in this PR.*

The padding is 16px horizontal, 24 px vertical on figma:
<img width="599" alt="Screenshot 2023-11-14 at 2 12 54 AM" src="https://github.com/crypto-power/cryptopower/assets/18400051/e829f3b0-3dd0-4d64-8a86-cc78c93a3a6c">

Then, there appears to be:
- 16px padding around the content of the expanded wallet view (the grey colored area below).
- 8px margin around each wallet item except the top of each item (the yellow colored area).

![Group 8855](https://github.com/crypto-power/cryptopower/assets/18400051/1d6f81b5-727e-458a-8b5a-69a88d9df763)

Reply:
https://github.com/crypto-power/cryptopower/issues/271